docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/index.md

```diff
@@ -0,0 +1,173 @@
+
+Optionally you may specify:
+- Alternative API Endpoint: To use a Data Structures API-compatible service that is not Snowplow BDP, you may specify its root API endpoint here.
+
+#### Iglu Server registries
+[Iglu Server](/docs/pipeline-components-and-applications/iglu/iglu-repositories/iglu-server/index.md) is a more full-featured dedicated service for hosting server that is more flexible than Static Registries.
+You may need this if you are integrating with [Snowplow Mini](/docs/pipeline-components-and-applications/snowplow-mini/index.md).
+
+To authenticate with your server, the extension will require:
+- Iglu API Endpoint: This is the base URL the extension will use when contacting the API. If you include a path component, the API request will be made as `api/*`, relative to this path; you may need to add or remove trailing slashes if the API is not hosted at the root.
+- Iglu API Key: See [API keys and the authentication service](/docs/pipeline-components-and-applications/iglu/iglu-repositories/iglu-server/index.md#5-api-keys-and-the-authentication-service-apiauth) for instructions on generating an Iglu API key.
+
+When logged into the Snowplow BDP console, these details should be available in [API keys for utilities](https://console.snowplowanalytics.com/iglu-keys) (within "Manage organization").
+
+#### Static registries
+Most other registries, such as those hosted as websites or via [S3](https://aws.amazon.com/s3/) or [GCS](https://cloud.google.com/products/storage/) buckets will use this  ["static"](/docs/pipeline-components-and-applications/iglu/iglu-repositories/static-repo/index.md) type.
+
+If you can access the schemas directly via a browser, this is probably the right choice.
+
+Because there is no API for listing the contained schemas, the extension will usually attempt to request any referenced schema from static registries to see if it includes it.
+To address this imbalance with the other registry types, [`igluctl`](/docs/pipeline-components-and-applications/iglu/igluctl-2/index.md) version 8 added support creating and uploading a file listing all the schemas in a registry.
+If the extension detects such a "manifest", it will successfully fetch and display schemas for you without events having to reference the schemas first.
+
+To connect to your static registry, the extension will need:
+- Base URI: The base URL to use when requesting schemas. The extension will request schemas relative to this base, expecting a `schemas/{vendor}/{name}/{format/{version}` format.
+
+Optionally you can also provide:
+- Manifest URI: The URI to use for the schema manifest (aka [Schema List](https://discourse.snowplow.io/t/igluctl-0-8-0-released/5551#2-uploads-schema-lists-by-default-2)) file that lists the schemas contained in the registry. If not provided, defaults to `schemas`, to match the `igluctl` default.
+
+## Events from other devices
```

Comment:
This feels like “advanced usage”. If we follow the suggestion to promote this to a higher level page, we can split this part out into its own page. (Similar to the Micro guide.)

Reply:
Again, here for completion's sake. Agreed it's out of place in a tutorial. Maybe wrap it in `<details>` or something until there's a better home for it?